File: Sc.Models/Dtos/ApproveRegistrations/Search/ApproveRegistrationHistorySearchDto.cs
using Sc.Models.Dtos.ApproveRegistrations.Base;

namespace Sc.Models.Dtos.ApproveRegistrations.Search
{
    public class ApproveRegistrationHistorySearchDto : BaseApproveRegistrationDto
    {
        public int ApproveRegistrationId { get; set; }
        public ApproveRegistrationHistoryFileDto File { get; set; }
    }
}

File: WebApplication/src/approve-registrations/dtos/search/approve-registration-history-search.dto.ts
import { ApproveRegistrationHistoryFileDto } from "../approve-registration-history-file.dto";
import { BaseApproveRegistrationDto } from "../base/base-approve-registration.dto";

export class ApproveRegistrationHistorySearchDto extends BaseApproveRegistrationDto {
    file: ApproveRegistrationHistoryFileDto;
    approveRegistrationId: number;
}